### PR TITLE
Prepare for 1.2.0 release

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5214)

### CODE REVIEW

**Comment by tgraf** [comment]
test-docs-please

**Comment by tgraf** [comment]
test-me-please

File: NEWS.rst [hunk 1]
```diff
@@ -2,377 +2,751 @@
...
+* Add metric "cilium_datapath_errors_total" for tracking errors in the datapath. (#4507, @joestringer)
+* Add Kafka specific CI test checks to make sure kafka cluster is up correctly. (#4488, @manalibhutiyani)
+* Metrics to report count of current endpoints tagged by endpoint states (#4376, @manalibhutiyani)
+* Use UpdateStatus for Cilium Network Policy Status in k8s 1.11 (#2972, @aanm)
+
+1.1.0
+=====
+
+    Amey Bhide (2):
+          contrib: Script to figure cilium pod for a given pod
```
Comment (aanm) [inline NEWS.rst]: The formatting doesn't look good

File: NEWS.rst [hunk 2]
```diff
@@ -2,377 +2,751 @@
...
+Major Changes
+-------------
 
-    André Martins (93):
-          examples/kubernetes: use POSIX regex for CILIUM_VERSION checker
-          docs: add checklinks target
-          docs: fix broken links
-          docs: use Documentation context to avoid longer image builds
-          docker/Dockerfile: update golang to 1.10.2
+* Add periodic aggregation of datapath notifications (#4936, @joestringer)
```
Comment (aanm) [inline NEWS.rst]: Can you replace the `'#` with `https://github.com/cilium/cilium/pull/`?

**Comment by tgraf** [comment]
Waiting on #5208 before addressing feedback

**Comment by nebril** [comment]
@aanm @tgraf should we change Cilium image version in `examples/kubernetes/addons/etcd-operator/cilium-development.yaml`?

**Comment by tgraf** [comment]
test-docs-please

**Comment by tgraf** [comment]
test-docs-please

**Pull request merged**